docs/protocol-methods.rst

```diff
@@ -1055,3 +1055,106 @@ Identify the client to the server and negotiate the protocol version.
 
   ["ElectrumX 1.2.1", "1.2"]
   "ElectrumX 1.2.1"
+
+masternode.announce.broadcast
+--------------
+
+Pass through the masternode announce message to be broadcast by the daemon.
+
+Whenever a masternode comes online or a client is syncing, they will send this message which describes the masternode entry and how to validate messages from it.
+
+**Signature**
+
+  .. function:: masternode.announce.broadcast(signmnb)
+
+  * *signmnb*
+
+    Signed masternode broadcast message.
```

Comment:
It's still not clear to me what this is like and you don't give any examples.  Can you do that in a follow-up?